package.json

```diff
@@ -77,11 +78,16 @@
     "mocha": "^8.2.0",
     "mocha-loader": "^5.1.5",
     "prettier": "^2.1.2",
-    "puppeteer": "^5.4.0",
+    "puppeteer": "^5.4.1",
     "style-loader": "^2.0.0",
     "url-loader": "^4.1.1",
     "webpack": "^4.0",
     "webpack-bundle-analyzer": "^3.9.0",
     "webpack-cli": "^4.1.0"
+  },
+  "dependencies": {
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
I don't think there should be any dependencies added here. Are these only test dependencies?